### ISSUE TITLE
IE11 Compatibility for v0.0.2?

### ISSUE THREAD

**Issue by lshiffer** [stellar/bifrost-js-sdk#8]
Hi,

I'm trying to run v0.0.2 on IE11....  From the source code I see comments about IE 11 and < 11 but getting an error after creating a session...

This:

```
var params = {
			network: 'test',
			horizonURL: HORIZON_URL,
			bifrostURL: BIFROST_URL,
			recoveryPublicKey: ISSUING_PUBLIC_KEY,
		};

		this.session = new Bifrost.Session(params);
```

Is leading to this:

> stack	"ReferenceError: 'Symbol' is undefined at _checkParams (http://localhost:3000/lib/bifrost-js-sdk/v0.0.2/bifrost.js:884:15)

From the IE debugger.

Any ideas?

Thank you.

**Comment by MisterTicot** [comment]
I got the same error in some webkit browsers when transpiling from JS6 using Babel. 
It came from the translation from 'for....of...' and my fix was to use older semantic instead. 
Alternatively you could include some content from babel-runtime but it seemed bloated to me.

**Comment by lshiffer** [comment]
Glad to know I'm not alone...

I've tried running the non-minified release through babel but get this error:

> Uncaught TypeError: Cannot read property 'TYPED_ARRAY_SUPPORT' of undefined

On both IE and Chrome and what not.

I'll try running with babel-runtime to see if that helps.

**Comment by lshiffer** [comment]
I think I got it to work by simply including babel-polyfill.  No transpiling or nothing, just including babel-polyfill before all the other scripts. 

I do wonder though if this is an oversight.  There are comments in the code about IE < 11 so the code should handle IE 11.  When walking through the debugger this all occurs when 'Symbol' is being used which isn't supported by IE.

**Comment by MisterTicot** [comment]
@Ishiffer
That's right, I remember, that's polyfill. The reason I didn't used that is it made my library like 1Mb larger. So I chose to simply rewrite my fews `for (let entry of array)...` loops into `for (let index in array) { const entry = array[index]...`. In my case there was no other construct depending on Symbol, if it even exist.
